### PR TITLE
Update release docs, add versioning

### PR DESCRIPTION
**Pull request by chapulina**
Documenting some of the things discussed in #76 .

@j-rivero , I'm not sure at what part of the release process the user should set the `IGNITION_VERSION` variable.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,19 +1,23 @@
...
-        * Blueprint and Citadel
-        * Dashing and Eloquent
-    * [ros2](https://github.com/osrf/ros_ign/tree/ros2)
-        * Citadel
-        * Foxy
+ROS version | Ignition version | Branch | Binaries hosted at
+-- | -- | -- | --
+Melodic | Blueprint | [melodic](https://github.com/osrf/ros_ign/tree/melodic) | https://packages.osrfoundation.org
+Melodic | Citadel | [melodic](https://github.com/osrf/ros_ign/tree/melodic) | :x:
+Noetic | Blueprint | :x: | :x:
```
Comment (j-rivero) [inline README.md]: If the branch does not support Blueprint in anyway we can remove it from the list or improve as  not supported somehow:

```suggestion
Noetic | Blueprint | not supported | :x:
```
same for Foxy below

**Review by j-rivero — APPROVED** [review]
minor thing to improve how non supported branches are listed but looks good anyway

File: README.md (hunk 1 above)
Comment (chapulina) [reply README.md]: a84dba439f1ea13742f9b4785cf959c73c1d3c74

**Comment by chapulina** [comment]
@j-rivero , do we need to add anything about `IGNITION_VERSION` to `RELEASING.md`? How can we assure we're releasing Melodic with Blueprint instead of Citadel?

**Comment by j-rivero** [comment]
> @j-rivero , do we need to add anything about `IGNITION_VERSION` to `RELEASING.md`? How can we assure we're releasing Melodic with Blueprint instead of Citadel?

You are totally right, we need to mention it at least the versions that go into `packages.osrfoundation.org`

**Comment by chapulina** [comment]
@j-rivero , do you mind adding it yourself? I actually don't know at which step to put it :sweat_smile:

**Comment by j-rivero** [comment]
> @j-rivero , do you mind adding it yourself? I actually don't know at which step to put it sweat_smile

Sure, 6e8c076

**Comment by chapulina** [comment]
Thanks. Let's merge this once https://github.com/ignition-tooling/release-tools/pull/249 is in.

**Pull request merged**